build-system/global-configs/experiments-config.json

```diff
@@ -1,5 +1,12 @@
 {
-  "experimentA": {},
+  "experimentA": {
+    "name": "MOVE_FIXED_LAYER",
+    "environment": "AMP",
+    "command": "gulp dist --define_experiment_constant=MOVE_FIXED_LAYER && gulp dist --esm --define_experiment_constant=MOVE_FIXED_LAYER",
```

Comment:
The experiment has the biggest benefit for documents on the origin (js file) but I want to make sure there's no adverse affect on documents on the cache (mjs) so I'd think we need to run it in both places . @erwinmombay you pointed this out, what do you think?

Reply:
What if you use `experimentA` for origin and `experimentB` for cache?

Reply:
We definitely need to do both, scs will try and inject the module/nomodule pattern

Reply:
Hmm. If this can wait for the weekend, can we chat early next week to discuss alternatives for the double build? As written, this PR will substantially increase Travis round trip time for all `master` builds, which will slow down development, releases, etc.

Reply:
I don't even think this will work correctly?

Reply:
> I don't even think this will work correctly?

@erwinmombay @jridgewell Did we ever get to the bottom of this? The bigger concern is that the main AMP release scripts also do the esm and non-esm minified compilations to the same `dist/` folder one after the other. So if this is broken, it follows that our releases will be broken too.

/cc @danielrozenberg